integration-tests/cypress/support/step_definitions/headings.js

```diff
@@ -0,0 +1,45 @@
+const { Then } = require('@badeball/cypress-cucumber-preprocessor')
+
+Then('I should see medium heading with text {string}', $string => {
+    cy.get('.govuk-heading-m').contains($string).should('exist')
+  })
+  
+  Then('I should see the heading {string}', $title => {
```

Comment:
Should this be `I should see the level 1 heading`?

Reply:
Yes... but:
This is not a new step, just moved over from the common steps file. Looking to obsolete this one in a follow up as it can be handled by `'I should a level {string} heading with text' below`